src/api/utils/handleResponseData.spec.ts

```diff
@@ -32,4 +32,4 @@ describe('handleResponseData()', () => {
             );
         });
     });
-});
\ No newline at end of file
+});
```

Comment:
I think we need to decide on trailing newlines vs no trailing newlines, and configure our editors consistently on this. Maybe that's something that prettier will help us with?

Reply:
Yeah prettier ftw - hopefully can get it done early after R30